pubtools/pulplib/_impl/model/convert.py

```diff
@@ -66,6 +66,17 @@ def tolerant_timestamp(value):
     return value
 
 
+def datetime_conversion(value):
```

Comment:
minor: modify the method name as "datetime_converter" or "timestamp_converter" to have a similar naming convention as other methods here.